com.unity.render-pipelines.universal/Runtime/ForwardRenderer.cs

```diff
@@ -748,5 +867,33 @@ bool CanCopyDepth(ref CameraData cameraData)
             bool msaaDepthResolve = false;
             return supportsDepthCopy || msaaDepthResolve;
         }
+
+        [Conditional("DEVELOPMENT_BUILD"), Conditional("UNITY_EDITOR")]
+        void SetupDebugRendering(ScriptableRenderContext context)
+        {
+            debugMaterialIndex = DebugDisplaySettings.Instance.materialSettings.DebugMaterialIndexData;
```

Comment:
Should we only set these if there's any debug mode active?

Reply:
This is now done differently and is only called when the debug-modes are active